psi4/driver/driver.py

```diff
@@ -872,6 +872,70 @@ def properties(*args, **kwargs):
     else:
         return core.variable('CURRENT ENERGY')
 
+def optimize_geometric(name, **kwargs):
+
+    # will move imports to top of file when i'm sure where this method goes
+    # TODO: check if GeomeTRIC is installed at the start
+    import qcengine as qcng
+    import qcelemental as qcel
+    from qcelemental.models import OptimizationInput
+
+    core.print_out('\n  Optimizing system with GeomeTRIC through QCEngine\n')
+
+    # TODO: return dummy wavefunction? Or run gradient at optimized geometry to get a wfn?
+    return_wfn = kwargs.pop('return_wfn', False)
+    if return_wfn:
+        raise ValidationError(f"return_wfn=True is not supported with GeomeTRIC.")
```

Comment:
of the two choices, I slightly favor running a final grad at the optimized geom to get a wfn

Reply:
I wonder if we should always run a final gradient (with or without a wavefunction request) in order to populate qcvars?